trunk/SUAVE/Components/Energy/Storages/Batteries/Constant_Mass/Lithium_Ion_LiNiMnCoO2_18650.py

```diff
@@ -203,11 +214,11 @@ def energy_cycle_model(self,numerics,discharge_flag = True ):
         n       = 1
         F       = 96485 # C/mol Faraday constant    
         delta_S = -496.66*(SOC_old)**6 +  1729.4*(SOC_old)**5 + -2278 *(SOC_old)**4 +  1382.2 *(SOC_old)**3 + \
-                  -380.47*(SOC_old)**2 + 46.508*(SOC_old) + -10.692  # eqn 10 and , D. Jeon Thermal Modelling ..
+                  -380.47*(SOC_old)**2 + 46.508*(SOC_old) + -10.692  
         
         i_cell         = I_cell/electrode_area # current intensity 
```

Comment:
From what I can tell the cell temperature math is the same between this battery and the other battery. Can you make a common function and add it to a higher level Lithium ion battery class?

If you need to update or you find a bug there's only one piece of code to fix

Reply:
Got it. Will change..